charts/admission-controller/templates/webhook/admissionregistration.yaml

```diff
@@ -1,3 +1,19 @@
+{{- if .Values.webhook.ssl.certManager.enabled }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "admissionController.webhook.fullname" . }}
+    helm.sh/hook: "post-install, post-upgrade"
```

Comment:
I'm not sure what the thinking is behind all the `$existingVac` logic, so I just skipped it here.